docs/fledge-rule-DataAvailability/index.rst

```diff
@@ -0,0 +1,15 @@
+.. Images
+.. |data-availability| image:: images/data-availability.png
+
+DataAvailability Rule
+=====================
+
+This is built in rule that triggers every time it receives data that matches the asset code or audit code is given.
+
++---------------------+
+| |data-availability| |
++---------------------+
+
+  - **Audit Code**: Audit log code to monitor, Leave blank if not required or set to * for all codes. If we want to monitor several audit codes we could enter a comma separated list e.g. SRVRG, SRVUN
```

Comment:
If we want to monitor several audit codes  a comma separated list can be entered. E.g. SRVRG, SRVUN